frontend/server/src/Controllers/Problem.php

```diff
@@ -6002,6 +6003,7 @@ public static function getCollectionsDetailsByLevelForTypeScript(\OmegaUp\Reques
             'rowcount'
         ) ?? \OmegaUp\Controllers\Problem::PAGE_SIZE;
         $difficulty = $r->ensureOptionalString('difficulty') ?? 'all';
+        $quality = $r->ensureOptionalString('quality') ?? 'all';
```

Comment:
```suggestion
        $quality = $r->ensureOptionalEnum('quality', self::QUALITY_VALUES) ?? 'all';
```

habría que poner un `const QUALITY_VALUES = ['onlyQualityProblems', 'all'];` arriba junto con las demás constantes.

igual en todos los demás lugares.

Reply:
Listo! :smile: